MunchkinLevelCounter/Util/Colors.swift

```diff
@@ -11,8 +11,8 @@ import UIKit
 
 struct Colors {
     static let black = UIColor.black
-    static let cardGeneral = UIColor(red: 0.97, green: 0.77, blue: 0.41, alpha: 1.0)
-    static let cardLight = UIColor(red:0.97, green:0.84, blue:0.45, alpha: 1.0)
-    static let cardCorner = UIColor(red:0.23, green:0.09, blue:0.02, alpha: 1.0)
-    static let cardBackground = UIColor(red:0.97, green:0.91, blue:0.83, alpha:1.0)
+    static let cardGeneral = UIColor(rgb: 0xF7D774)
```

Comment:
It's some old commit again. An argument was misnamed. Now it's hex: which is more suitable name for it. In general I have added this extension because I think that the representation of color as hex is more familiar and more compact.